src/allocator.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+This is a part of Loops project.
+Distributed under Apache 2 license.
+See https://github.com/vpisarev/loops/LICENSE
+*/
+
+#include "allocator.hpp"
+#include <string>
+#include <unistd.h>
```

Comment:
unistd.h is unix-specific include, please, surround it with proper ifdefs and report error when it's compiled under Windows (which is unlikely but possible)

Reply:
Done.